setup-cluster.sh

```diff
@@ -26,23 +39,23 @@ cat << EOF | tee ${VAGRANT_CONFIG}
 \$node_count = ${NODE_COUNT}
 \$master_count = 1
 
-\$vm_memory = 1024
-\$vm_vcpus = 1
-\$vm_disk = 45
+\$vm_memory = 16192
```

Comment:
@rthallisey where did you see the 500MB requirement, the pre-reqs I'm aware of are here:
https://docs.openshift.org/latest/install_config/install/prerequisites.html

Masters: 16GB
Worker Nodes: 8GB

We've tested with lower requirements and have been OK in past, but never tried anything as low as 500MB